common/protob/messages-cardano.proto

```diff
@@ -500,3 +500,49 @@ message CardanoTxBodyHash {
  */
 message CardanoSignTxFinished {
 }
+
+/**
+ * Request: Ask device to sign a message containing arbitrary data
+ * @start
+ * @next CardanoMessageItemAck
+ */
+message CardanoSignMessageInit {
+    required uint32 protocol_magic = 1;                           // network's protocol magic
```

Comment:
What's the purpose of this? I don't think we are doing anything with Byron addresses for message signing. And isn't network id part of `CardanoAddressParametersType`? (If not, why is it required while the address is optional?)

Reply:
@janmazak Network id isn't part of CardanoAddressParametersType, but you are right in that protocol magic and network id should be as optional as address parameters.

As for byron address, our discussion about them in slack hasn't reached a conclusion yet, please chime in